overlay.d/05rhcos/usr/lib/dracut/modules.d/40rhcos-fips/rhcos-fips.sh

```diff
@@ -69,6 +69,17 @@ finish() {
         fatal "FIPS mode is not enabled."
     fi
 
+    # If we're running from a live system, then set things up so that the dracut fips
+    # module will find the kernel binary.  TODO change dracut to look in /usr/lib/modules/$(uname -r)
+    # directly.
+    if test -f /etc/coreos-live-initramfs; then
+        # See the dracut source
+        rhevh_livedir=/run/initramfs/live
+        mkdir -p "${rhevh_livedir}"
+        # Why "vmlinuz0"?  I have no idea; it's what the dracut fips module uses.
+        ln -sr /usr/lib/modules/$(uname -r)/vmlinuz ${rhevh_livedir}/vmlinuz0
+    fi
+
```

Comment:
Since for testing purposes there is a workaround for this bit, I'd be inclined to defer this to another PR and try to get the --rbind thing merged.

Reply:
With just the first commit, all that happens in FIPS mode is the system dies in boot in a different way...doesn't seem useful?

Reply:
> Since for testing purposes there is a workaround for this bit

Oh, are you thinking of the `rd.fips.skipkernel` kernel argument?  Yes true...